chia/util/config.py

```diff
@@ -177,23 +180,24 @@ def traverse_dict(d: Dict, key_path: str) -> Any:
 def process_config_start_method(
     config: Dict[str, Any],
     log=logging.Logger,
-) -> Optional[Literal["fork", "forkserver", "spawn"]]:
-    from_config = config.get("multiprocessing_start_method")
+) -> method_values:
+    from_config: object = config.get("multiprocessing_start_method")
```

Comment:
Why not do 

```
from_config = config.get("multiprocessing_start_method", "default")
```

Reply:
If we allow `multiprocessing_start_method:` or ``multiprocessing_start_method: null` then we have to handle `None` anyways.  We could drop handling that though, sure.  It's probably more useful with lists like the pools.